### PR TITLE
Update index page links based on journeys

### LINKED ISSUE #9: Update links on the index page
Action links on the index page should point to the correct doc section.

### PR DESCRIPTION
**Pull request by oraNod**
This PR fixes #9 improves the yaml for the index page based on persona journeys.

### CODE REVIEW

File: data/pages.yaml [hunk 1]
```diff
@@ -1,51 +1,67 @@
...
-    action_one: Understand the fundamentals of Ansible automation
-    action_two: Set up an Ansible automation project
-    action_three: Write your first playbook in a few easy steps
-  #Ansible installation
-  install_action: Go to the installation guide
-  #Ansible user
+    # Labels provide meaningful descriptions of elements for accessibility purposes.
+    label: Button group that links to documentation for new Ansible users.
+    learn: Understand the fundamentals of Ansible automation
+    setup: Set up an Ansible automation project
```
Comment (samccann) [inline data/pages.yaml:24]:
```suggestion
    setup: Install the Ansible package
```

Comment (samccann) [reply data/pages.yaml:24]: The original text doesn't feel right to me. This is specifically only dealing with ansible package install, not setting up an automation project as a whole. When I see the original text, I think it's doing something more (and potentially scary as a new user).

File: data/pages.yaml [hunk 2]
```diff
@@ -1,51 +1,67 @@
...
-    action_one: Set up your development environment
-    action_two: Review the Ansible contributor guidelines
-    action_three: Write custom modules or plugins
+  developer_body: Ansible is open source. You can add code that fixes bugs, implements new features, or extends the benefits of simple, agentless automation.
+  developer_actions:
+    # Labels provide meaningful descriptions of elements for accessibility purposes.
+    label: Button group that links to base Ansible developer documentation.
+    setup: Set up your development environment
+    guidelines: Review the Ansible contributor guidelines
+    start: Write custom modules or plugins
```
Comment (samccann) [inline data/pages.yaml:48]: looks like the `learn` is missing here? There's no longer a link to the architecture showing up tho it is listed at https://github.com/ansible/jinja-docsite/pull/33/files#diff-e63e46cde797a05be9a5a69b973f28a8c8d68cb2b161a39f817dd8c637af0facR22 (links.yaml)

File: data/pages.yaml (hunk 1 above)
Comment (oraNod) [reply data/pages.yaml:24]: Yeah I think I put "set up automation project" as a gap but didn't alter the text here: https://github.com/ansible/jinja-docsite/issues/32

**Comment by oraNod** [comment]
@samccann I incorporated your change for "Install the Ansible package" but ideally that will be in place until we can update content and close the gap for the "setting up an environment" action, if that makes sense.

**Pull request merged**